irods/test/scripts/test002.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+. "$BATS_TEST_DIRNAME"/funcs
+PYTHON=python3
+
+# Setup/prerequisites are same as for login_auth_test.
+# Run as ubuntu user with sudo; python_irodsclient must be installed (in either ~/.local or a virtualenv)
+#
+
+PASSWD=test123
+NEW_PAM_PASSWD="new_pass"
```

Comment:
It's not clear whether these variables are OS or iRODS specific.

I'm guessing that `NEW_PAM_PASSWD` is not known to the OS. However `PASSWD` feels like it could be known to the OS (i.e. special).

Perhaps we add a prefix to remove any assumptions. Thoughts?

Reply:
Probably a good idea. Will do it.

Reply:
Will use OS_ for now, pending final review.

Reply:
OK actually in these functions they blur a bit so I'll leave prefixes off for now.

Reply:
The way I figure it, since in bash, a variable assignment doesn't affect subshells unless exported, it's not a danger that these would bleed over into any commands executed by this script.

But, how about we just call them ALICE_NEW_PAM_PASSWORD and ALICE_OLD_PAM_PASSWORD?

Reply:
ok I think this is done!